arwen/elrondapi/elrondei.go

```diff
@@ -1142,6 +1146,68 @@ func v1_3_transferESDTNFTExecute(
 		dataOffset)
 }
 
+//export v1_3_multiTransferESDTNFTExecute
+func v1_3_multiTransferESDTNFTExecute(
+	context unsafe.Pointer,
+	destOffset int32,
+	numTokenTransfers int32,
+	tokenTransfersArgsLengthOffset int32,
+	tokenTransferDataOffset int32,
+	gasLimit int64,
+	functionOffset int32,
+	functionLength int32,
+	numArguments int32,
+	argumentsLengthOffset int32,
+	dataOffset int32,
+) int32 {
+	host := arwen.GetVMHost(context)
+	runtime := host.Runtime()
+	metering := host.Metering()
+
+	callArgs, err := extractIndirectContractCallArgumentsWithoutValue(
```

Comment:
Optionally return early if `numTokenTransfers == 0`, but consuming the gas cost of one ESDTTransfer.

Reply:
done